addons/l10n_uy/data/l10n_latam.document.type.csv

```diff
@@ -1,52 +1,52 @@
-"dc_e_boleta",151,"e-Boleta","invoice","e-BO","base.uy"
-"dc_nota_de_credito_e_boleta",152,"Credit Note e-Boleta","credit_note","e-BO-NC","base.uy"
-"dc_nota_de_debito_e_boleta",153,"Debit Note e-Boleta","debit_note","e-BO-ND","base.uy"
-"dc_e_remito",181,"e-Delivery Guide",,"e-REM","base.uy"
-"dc_e_resguardo",182,"e-Resguardo",,"e-RES","base.uy"
-"dc_e_ticket_cont",201,"Contingency e-Ticket","invoice","e-TK-C","base.uy"
-"dc_cn_e_ticket_cont",202,"Contingency e-Ticket Credit Note","credit_note","e-NCTK-C","base.uy"
-"dc_dn_e_ticket_cont",203,"Contingency e-Ticket Debit Note","debit_note","e-NDTK-C","base.uy"
-"dc_e_factura_contingencia",211,"Contingency e-Invoice","invoice","e-FC-C","base.uy"
-"dc_nota_de_crédito_de_e_factura_contingencia",212,"Contingency e-Invoice Credit Note","credit_note","e-NC-C","base.uy"
-"dc_nota_de_débito_de_e_factura_contingencia",213,"Contingency e-Invoice Debit Note","debit_note","e-ND-C","base.uy"
-"dc_e_factura_exportación_contingencia",221,"Contingency Export e-Invoice","invoice","e-FCE-C","base.uy"
-"dc_nota_de_crédito_de_e_factura_exportación_contingencia",222,"Contingency Export e-Invoice Credit Note","credit_note","e-NCE-C","base.uy"
-"dc_nota_de_débito_de_e_factura_exportación_contingencia",223,"Contingency Export e-Invoice Debit Note","debit_note","e-NDE-C","base.uy"
-"dc_e_remito_de_exportación_contingencia",224,"Contingency Export e-Delivery Guide",,"e-REME-C","base.uy"
-"dc_e_ticket_venta_por_cuenta_ajena_contingencia",231,"Contingency e-Ticket Sale By Third Party","invoice","e-TK-CAC","base.uy"
-"dc_nota_de_crédito_de_e_ticket_venta_por_cuenta_ajena_contingencia",232,"Contingency e-Ticket Sale By Third Party Credit Note","credit_note","e-NCTK-CAC","base.uy"
-"dc_nota_de_débito_de_e_ticket_venta_por_cuenta_ajena_contingencia",233,"Contingency e-Ticket Sale By Third Party Debit Note","debit_note","e-NDTK-CAC","base.uy"
-"dc_e_factura_venta_por_cuenta_ajena_contingencia",241,"Contingency e-Invoice Sale By Third Party","invoice","e-FC-CAC","base.uy"
-"dc_nota_de_crédito_de_e_factura_venta_por_cuenta_ajena_contingencia",242,"Contingency e-Invoice Sale By Third Party Credit Note","credit_note","e-NC-CAC","base.uy"
-"dc_nota_de_débito_de_e_factura_venta_por_cuenta_ajena_contingencia",243,"Contingency e-Invoice Sale By Third Party Debit Note","debit_note","e-ND-CAC","base.uy"
-"dc_e_boleta_contingencia",251,"Contingency e-Boleta","invoice","e-BO-C","base.uy"
-"dc_nota_de_credito_e_boleta_contingencia",252,"Contingency e-Boleta Credit Note","credit_note","e-BO-NC-C","base.uy"
-"dc_nota_de_debito_e_boleta_contingencia",253,"Contingency e-Boleta Debit Note","debit_note","e-BO-ND-C","base.uy"
-"dc_e_remito_contingencia",281,"Contingency e-Delivery Guide",,"e-REM-C","base.uy"
-"dc_e_resguardo_contingencia",282,"Contingency e-Resguardo",,"e-RES-C","base.uy"
+"id","code","name","internal_type","doc_code_prefix","country_id/id","name@es","name@es_419"

```

Comment:
Same

Reply:
Should be `es_419` only as well. `es` is only for Spain.